spec/requests/http_basic_auth_spec.rb

```diff
@@ -7,6 +7,16 @@
     APP_CONFIG.http_auth_password = "secret"
   end
 
+  before(:each) do
+    @fqdn = "market.custom.org"
```

Comment:
That's quite cryptic variable name... What does it mean? Could it be changed to something more clear?